common/views/components/PageHeader/PageHeader.tsx

```diff
@@ -63,30 +48,20 @@ const HeroPictureContainer = styled.div`
   `}
 `;
 
-export type FeaturedMedia =
-  | Element<typeof UiImage>
-  | Element<typeof VideoEmbed>
-  | Element<typeof Picture>;
+export type FeaturedMedia = UiImage | VideoEmbed | Picture;
 
-type BackgroundType = Element<typeof HeaderBackground>;
+type BackgroundType = ReactElement<typeof HeaderBackground>;
 
 export function getFeaturedMedia(
   fields: GenericContentFields,
   isPicture?: boolean
-): ?FeaturedMedia {
+): FeaturedMedia | undefined {
   const image = fields.promo && fields.promo.image;
   const { squareImage, widescreenImage } = fields;
   const { body } = fields;
-  const tasl = image && {
-    title: image.title,
```

Comment:
I stripped this out because none of these fields exist on `image` – they exist on `image.tasl`, but that's already being passed to `UiImage` so we don't need to repeat it here. As far as I can tell, this value would always be an empty object.